option optimize_for = LITE_RUNTIME;

message VipWeekPackageBuyRequest
{
 required uint32 id = 1;
}
